scripts/lib/install

```diff
@@ -14,16 +14,17 @@ DEPLOYMENT_TYPE="${DEPLOYMENT_TYPE:-voyager}"
 # Docker.  Use e.g. zulip::app_frontend for a Zulip frontend server.
 PUPPET_CLASSES="${PUPPET_CLASSES:-zulip::voyager}"
 VIRTUALENV_NEEDED="${VIRTUALENV_NEEDED:-yes}"
+ZULIP_ROOT="$(realpath $(dirname $0)/../../)"
```

Comment:
It's not nedded the "/" character in the end, this generate duplicate to concatenate.
Rename `ZULIP_ROOT` to `ZULIP_PATH` for mantain the same name used in Python scripts.